js/build-env-config.js

```diff
@@ -12,10 +12,16 @@ if (environment !== `PRODUCTION`) {
   habitat.load(`config/default.env`);
 }
 
-var config = {
+const HOST = habitat.get(`HOST`);
+const PORT = habitat.get(`PORT`);
+const ORIGIN = HOST + (PORT ? `:${PORT}` : ``);
+const CONFIG = {
```

Comment:
ES6 for the win:
```
const CONFIG = { HOST, PORT, ORIGIN, PULSE_API: habitat.get(`PULSE_API`)};
```
This is the same as what you wrote, but ES6 improved object notation allows you to use `x` for any object propert:value pair `x:x`. Super convenient.